src/OpenTelemetry/Metrics/Aggregators/Aggregator.cs

```diff
@@ -18,15 +18,34 @@
 
 namespace OpenTelemetry.Metrics.Aggregators
 {
+    /// <summary>
+    /// Aggregator base class.
+    /// </summary>
+    /// <typeparam name="T">The type.</typeparam>
     public abstract class Aggregator<T>
         where T : struct
     {
+        /// <summary>
+        /// Adds value to the running total in a thread safe manner.
+        /// </summary>
+        /// <param name="value">Value to be added to running total.</param>
         public abstract void Update(T value);
 
+        /// <summary>
+        /// Checkpoints the current running sum into checkpoint, and starts counting again.
```

Comment:
we cannot be sure that the aggregate data is sum or not. So i'd refrain from making assumptions. We can simply state "the current aggregate data".. "resets the state"